db/migrate/20220805192937_fix_invalid_transfers.rb

```diff
@@ -0,0 +1,7 @@
+class FixInvalidTransfers < ActiveRecord::Migration[7.0]
+  def change
+    Transfer.all.each do |transfer|
+      transfer.destroy if transfer.from.nil? || transfer.to.nil?
+    end
```

Comment:
Thanks for digging into this! I'll go ahead and run this in production so we don't need this.